content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: "From Code to Quill: Embark on a Legendary Kubernetes Quest with SIG-Docs"
+date: 2024-04-28
+slug: contribute-as-sig-docs-reviewer
+author: >
+   Ricardo Amaro (Acquia)
+---
+
+You've probably come across the saying, "Contributing doesn't always mean writing lines of code"
+whispered to the wind or glimpsed in the margins of a sacred open-source tome.
+What is the truth behind this tale as old as time (or at least as old as Kubernetes itself)?
+
+Imagine Sarah: an enthusiast of open-source whose keyboard had seen more action crafting
+technical masterpieces for her projects than a knight's sword in battle. Sarah's had such
+a smooth relationship with Git, that she could `git commit` and `git push` in her sleep,
+and her etiquette in pull-requests was so polished, it could make a silver spoon look tarnished.
```

Comment:
```suggestion
and her etiquette in pull requests was so polished, it could make a silver spoon look tarnished.
```

pull requests as a term is not hyphenated